index.js

```diff
@@ -163,7 +164,7 @@ function parseStyles(
 ) {
   var statements = parseStatements(result, styles)
 
-  return Promise.all(statements.map(function(stmt) {
+  return Promise.resolve(statements).then(promiseEach(function(stmt) {
```

Comment:
what is the difference between promise each and Promise.all? Cause from [the doc](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Promise/all) I can see

>  The array of values maintains the order of the original iterable object, not the order that the promises were resolved in.